src/api/spec/controllers/webui/patchinfo_controller_spec.rb

```diff
@@ -138,15 +138,27 @@ def do_proper_post_save
   end
 
   describe 'GET #show' do
-    before do
-      login user
-      get :show, params: { project: user.home_project_name, package: patchinfo_package.name }
+    context 'package does not exist' do
+      before do
+        login user
+        get :show, params: { project: user.home_project_name, package: 'foo' }
+      end
+
+      it { expect(flash[:error]).to eq("Patchinfo 'foo' not found in project '#{user.home_project_name}'") }
+      it { expect(response).to have_http_status(:redirect) }
     end
 
-    it { expect(response).to have_http_status(:success) }
-    it { expect(assigns(:binaries)).to be_a Array }
-    it { expect(assigns(:pkg_names)).to be_empty }
-    it { expect(assigns(:packager)).to eq(user) }
+    context 'project and package exist' do
+      before do
+        login user
+        get :show, params: { project: user.home_project_name, package: patchinfo_package.name }
+      end
+
+      it { expect(response).to have_http_status(:success) }
+      it { expect(assigns(:binaries)).to be_a Array }
```

Comment:
same here.

Reply:
@coolo  what do you think here? ```it { expect(assigns(:binaries)).to be_a Array }``` ? is that better like?  ```it { expect(assigns(:binaries)).to be_a(Array) } ``` ?